CDP4Composition/Reporting/ICDP4ObjectDataSource.cs

```diff
@@ -0,0 +1,9 @@
+﻿using CDP4Common.EngineeringModelData;
+
+namespace CDP4Composition.Reporting
+{
+    public interface ICDP4ObjectDataSource
```

Comment:
File missing header and docs

Reply:
These files should not even be here any more, as they are leftovers from the initial demo implementation, which are no longer relevant.